Sources/FluentUI_iOS/Components/Bottom Sheet/BottomSheetController.swift

```diff
@@ -1130,8 +1165,8 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
     // Height of the sheet in collapsed state
     private var collapsedSheetHeight: CGFloat {
         let safeAreaSheetHeight: CGFloat
-        if resolvedCollapsedSheetHeight > 0 {
-            safeAreaSheetHeight = resolvedCollapsedSheetHeight
+        if let dynamicHeight = resolvedDynamicSheetHeights?.collapsedHeight, dynamicHeight > 0  {
```

Comment:
super minor: extra space between `0` and `{`

Reply:
Don't mind me, just making a suggestion so github gives me the nice button to make the change directly.

```suggestion
        if let dynamicHeight = resolvedDynamicSheetHeights?.collapsedHeight, dynamicHeight > 0 {
```

Reply:
it worked! Nice.